Class/Practise 4.1/src/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include "PizzaBuilder.h"
+#include "IngredientsReader.h"
+#include <iostream>
+#include <string>
+#include <print>
+
+namespace {
```

Comment:
Like for named constants. also they can be constexpr which in some cases may impact performance